Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/package.json

```diff
@@ -27,6 +28,7 @@
   "dependencies": {
     "axe-core": "^4.5.2",
     "cypress-sql-server": "^1.0.0",
+    "dotenv": "^16.3.1",
```

Comment:
dotenv has been added but I don't see it being used anywhere yet? Is this planning to be added in the future?

Reply:
Hey Nick,
That’s correct.  I’ve added it in case we want to set environment variables in a .env file if there are any values we want from the CI pipeline if we want to run the tests from the repo locally on our machine, or of course we want to run the cypress commands with the environwment variables set in the .env file rather than set as arguments in the command-line upon test execution.